ads/_config.js

```diff
@@ -396,7 +396,9 @@ export const adConfig = {
     prefetch: 'https://amp.ad.dotandad.com/dotandadsAmp.js',
     preconnect: 'https://bal.ad.dotandad.com',
   },
-
+  'dynad': {
+    preconnect: ['https://t.dynad.net', 'https://tm.jsuol.com.br'],
```

Comment:
So I noticed here you are not implementing `renderStartImplemented`. If possible, could you please implement this? It helps us serve ads as fast as possible to our users, and gives us some big performance gains.

Though, from your PR description, I'm not sure if you are making the PR on `dynad`'s behalf, can can implement that server side for them?

Let me know what you think. Thank you! 😄